react/features/base/react/components/web/MultiSelectAutocomplete.js

```diff
@@ -134,7 +135,7 @@ class MultiSelectAutocomplete extends Component {
             isOpen: Boolean(this.state.items.length)
                 && Boolean(filterValue),
             items: filterValue ? this.state.items : []
-        }, this._sendQuery);
+        }, debounce(this._sendQuery, 200));
```

Comment:
I've noticed the same thing after lunch actually and am currently fixing it. I'll update the pr later this afternoon.